build-tools/packages/build-cli/src/commands/info.ts

```diff
@@ -22,6 +62,18 @@ export default class InfoCommand extends BaseCommand<typeof InfoCommand> {
 		releaseGroup: releaseGroupFlag({
 			required: false,
 		}),
+		columns: Flags.option({
+			char: "c",
+			description: "Specify which columns are included in report.",
+			aliases: ["columns"],
```

Comment:
Did you intend to use a different string for the alias? Maybe "column"?

Reply:
Haha...  no.  I just didn't know how oclif worked and thought you used the 'aliases' field to specify an alternative name for the 'char'.  (Removed)